src/plugins/data/common/search/session/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export * from './status';
+export * from './types';
+
+export const SEARCH_SESSIONS_TABLE_ID = 'searchSessionsMgmtUiTable';
```

Comment:
I didn't change this, just moved data_enhanced/common -> data/common

But _I assume_ because this is used in functional tests are really running on a _server_ 
so this makes sense in some way